docs/kind_kafka_cluster.md

```diff
@@ -68,7 +68,7 @@ data:
 
 ```
 Specify Strimzi Kafka Version (See... https://github.com/strimzi/strimzi-kafka-operator/releases)
-export STRIMZI_VERSION=0.18.0
+export STRIMZI_VERSION=0.22.1
```

Comment:
you might run into the following with that version:
```
The CustomResourceDefinition "kafkas.kafka.strimzi.io" is invalid: metadata.annotations: Too long: must have at most 262144 bytes
```

see here https://github.com/matzew/kn-box/commit/33d81c2d0964360f4cd5e730eadad8673bb32a1e#diff-2c73b452095ee223ceb2ca2dedbb5a6d59e2e2e32140d5bc71fdd762c96d31f7

Reply:
I was naïvely pulling sample versions along as I found them in the readme; would 0.21.0 be a better example to leave here for now?

Reply:
I've changed the example to use 0.21.1